notes/20200217_meeting.md

```diff
@@ -39,6 +39,9 @@ Other benefits of RAP include:
 ### Increased trust:
 - code and documentation open to anyone (if possible)
 - software open to anyone?
+- know who has edited code, when, what changed and why
+- unit tests and peer review provide reassurance that the code works as expected
+- reduced chance of errors compared to a more traditional spreadsheet-based approach
```

Comment:
So are we saying spreadsheets can't be used in a RAP?

Reply:
Personally, I would see the spreadsheets only being used as inputs and outputs of a RAP but happy for others to challenge me there! I guess I was trying to compare to a more traditional approach of producing publications using a series of interlinked spreadsheets and copy and pasting vs a RAP type approach

Reply:
I think I'd agree. I see the move away from spreadsheets (outside of I/O) as minimum RAP (but not MVP).

Reply:
Yes I think I agree, just wanted to clarify. Jamie, can I check what you mean by 'minimum RAP' and 'MVP'? To me these are the same thing...?

Reply:
Minimum RAP: Moving to a coding language (e.g from Excel). That's it. Basically RAP level 1.

Minimum Viable RAP: A level which all projects should be striving for at a minimum (defined by us). It should be better than just using Python or R, incorporating some best practices, peer review, etc.

Reply:
Got it, thanks!

Reply:
Could you count VBA in RAP?

Reply:
Given how tightly couple VBA is to Excel, I'd be inclined to say no. Someone at Dstl has written an article on our internal Wiki which goes into detail at just how bad Excel is. I don't think we should be encouraging it (but then I have a fairly hard stance on this issue).

Reply:
I agree! I think VBA is harder to inspect than programming languages, which is a distinct disadvantage, but I would also be interested in reading that article if you're able to share it??

Reply:
http://www.eusprig.org/horror-stories.htm

Realistically, you can't easily make automated pipelines in one programme with Excel Spreadsheets. But I know that there is going to be some VBA don who will disagree.